spec/views/application_view_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationView, type: :view do
+  it { is_expected.to be_a(Stache::Mustache::View) }
+
+  describe '#i18n' do
+    subject { described_class.new.i18n }
+    it { is_expected.to be_instance_of(View::Translator) }
+
+    it 'sets the context as the translator scope' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.